src/main/java/net/atos/zac/app/informatieobjecten/model/RESTInformatieObjectZoekParameters.java

```diff
@@ -17,4 +17,6 @@ public class RESTInformatieObjectZoekParameters {
     public UUID[] UUIDs;
 
     public boolean toonGekoppeldeZaakDocumenten;
+
+    public UUID[] toegestaneInformatieObjectTypeUUIDs;
```

Comment:
Geef hier het betreffende besluittype terug (besluittypeUUID) ipv in de front-end te bepalen wat de informatieobjecttypen zijn die bij het besluittype horen

Reply:
Opgelost in opvolgende commit